mkosi/distributions/gentoo.py

```diff
@@ -196,67 +184,53 @@ def __init__(
             "--usepkg=y",
             "--keep-going=y",
             f"--jobs={jobs}",
-            f"--load-average={jobs-1}",
+            f"--load-average={jobs+1}",
             "--nospinner",
         ]
         if "build-script" in ARG_DEBUG:
-            self.emerge_default_opts += ["--verbose",
-                                         "--quiet=n",
-                                         "--quiet-fail=n"]
+            self.emerge_default_opts += ["--verbose", "--quiet=n", "--quiet-fail=n"]
         else:
             self.emerge_default_opts += ["--quiet-build", "--quiet"]
 
         self.arch, _ = ARCHITECTURES[state.config.architecture or "x86_64"]
-
-        #######################################################################
-        # GENTOO_UPSTREAM : we only support systemd profiles! and only the
-        # no-multilib flavour of those, for now;
-        # GENTOO_UPSTREAM : wait for fix upstream:
-        # https://bugs.gentoo.org/792081
-        #######################################################################
-        # GENTOO_DONTMOVE : could be done inside set_profile, however
-        # stage3_fetch() will be needing this if we want to allow users to pick
-        # profile
-        #######################################################################
-        self.arch_profile = Path(f"profiles/default/linux/{self.arch}/{state.config.release}/systemd")
-
-        self.pkgs_sys = ["@world"]
-
-        self.pkgs_fs = ["sys-fs/dosfstools"]
-
-        if not state.do_run_build_script and state.config.bootable:
-            self.pkgs_boot += ["sys-kernel/installkernel-systemd-boot",
-                               "sys-kernel/gentoo-kernel-bin",
-                               "sys-firmware/edk2-ovmf"]
+        self.arch_profile = Path(f"default/linux/{self.arch}/"
+                                 f"{state.config.release}/no-multilib/systemd/merged-usr")
+        self.pkgs['sys'] = ["@world"]
+
+        self.pkgs['fs'] = ["sys-fs/dosfstools"]
+        if state.config.output_format == OutputFormat.subvolume:
+            self.pkgs['fs'] += ["sys-fs/btrfs-progs"]
+        elif state.config.output_format == OutputFormat.disk:
+            self.pkgs['fs'] += ["sys-fs/btrfs-progs"]
+            self.pkgs['fs'] += ["sys-fs/xfsprogs"]
+            self.pkgs['fs'] += ["sys-fs/e2fsprogs"]
```

Comment:
We should drop this, we don't install any fs tools by default for other distros anymore either

Reply:
even `btrfs-progs` and `dosfstools`?

Reply:
Yup, because with systemd-repart, we don't know what filesystems will be in the final image, so we don't add anything by default, and leave it up to the user.

Reply:
we do if output is subvolume! unless i'm misunderstood what that means.

Reply:
removing `dostfstools`

Reply:
removed `btrfs-progs` too but i would be interested to know more about the case  `output == subvolume`.

Reply:
In case of `output == subvolume`, root directory is indeed a subvolume but that doesn't mean that the user won't pack up the subvolume inside an ext4 filesystem or something else later, just because the root directory is a subvolume does not mean that the user will want to have btrfs-progs available inside the root directory

Reply:
ok! but now i would like to meet that user, lol.
well actually i just don't know a use case like that but i get that we need to stay as generic as possible.